lib/inferred_concepts/applier.rb

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module InferredConcepts
+    class Applier
```

Comment:
Methods from this class that need a persona could be instance methods, and the persona could be passed in an initializer. DI makes it easier to test.